packages/projects-docs/pages/learn/editors/_meta.json

```diff
@@ -1,6 +1,9 @@
 {
   "overview": "Overview",
-  "web": "Web",
-  "ios": "iOS",
-  "vscode": "VS Code"
+  "web-interface": "Web interface",
+  "desktop-interface": "Desktop interface",
+  "desktop-setup": "Desktop setup",
+  "shortcuts": "Keybindings",
+  "settings": "Settings",
+  "themes": "Themes"
```

Comment:
Should we drop the Themes page @necoline?

Reply:
we should, your are right